docs/destination.md

```diff
@@ -92,3 +92,102 @@ The Timeplus source plugin for Airbyte is in the early stage. Please contact us
 ## Trigger Actions via webhook{#webhook}
 
 You can also add automations to trigger other systems to take actions when Timeplus finds any real-time insights. Simply choose the **Webhook** as the action type and optionally set a message body (by default, the entire row will be encoded as a JSON document and sent to the webhook). You can use this approach to perform rule-based automation without human interaction, such as swapping a overheated equipment, scaling up to scaling down the server farm, or reminder users on slack, etc. Please check [this blog](https://www.timeplus.com/post/build-a-real-time-security-app-in-3-easy-steps) for real-world examples.
+
+
+## Sink API
+In case calling API to create a sink, here are the reference
+
+### kafka
```

Comment:
might be okay to use `kafka` as the source type in the API. Ideally should be Apache Kafka

Reply:
here the name is same as what we call in the api